internal/output/preamble.go

```diff
@@ -19,7 +19,7 @@ var PreambleTimestamp time.Time
 var PreambleCreator string
 
 // Preamble returns file auto-generated preamble with specified comment character.
-func Preamble(commentPrefix string) string {
+func Preamble(commentPrefix string, commentPostFixes ...string) string {
```

Comment:
Made this variadic to make the postfix optional. Don't know of a cleaner way other than introducing functional opts. Happy to make another PR for that.